src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryDate.cs

```diff
@@ -354,12 +355,12 @@ public static FormulaValue Time(IRContext irContext, NumberValue[] args)
             return new TimeValue(irContext, result);
         }
 
-        private static FormulaValue Now(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, FormulaValue[] args)
+        private static async ValueTask<FormulaValue> Now(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, FormulaValue[] args)
         {
             return new DateTimeValue(irContext, DateTime.Now);
         }
 
-        public static FormulaValue DateParse(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, StringValue[] args)
+        private static async ValueTask<FormulaValue> DateParse(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, StringValue[] args)
```

Comment:
Hi Mike, is this supposed to be private now? Isn't this function targeted by the DateValue function?

Reply:
DateValue is in this class, has access to this private static function.

Reply:
Actually, pretty much all these functions in Library could be private.

Reply:
This instance is just a copy&paste error (I had *a lot* of mechanical changes to make ;) )

The class is non-public, so these methods all effectively private anyways.

Reply:
Still private/public reads odd - can we normalize to one or another? Think of the "next guy" trying to infer hidden meaning from the difference :)